### PR TITLE
(PAYM-1913) - Update action to reduce complexity.

### PR DESCRIPTION
**Pull request by nbenmoody-tesouro**
Motivation
I want to reduce the complexity of this action, to make it more flexible for use.

Modifications
I removed all the editing, which will need to be picked up by the microservices as needed, using a combination of `yq` and env vars.

### CODE REVIEW

File: armory-deploy/example-job.yaml [hunk 1]
```diff
@@ -2,23 +2,28 @@
...
+    needs: [ci-get-version, ci-functional-tests]
     steps:
       - name: Checkout
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
+        # https://github.com/mikefarah/yq
+      - name: Update Version
+        uses: mikefarah/yq@master
```
Comment (jonathan-Ts) [inline armory-deploy/example-job.yaml:16]: Would it be better to use a tagged version instead of a branch for third party code?

Comment (Tesouro-Chris) [reply armory-deploy/example-job.yaml:16]: Ditto. Recommending `v4.29.2`

**Review by Tesouro-Chris — COMMENTED** [review]
Other than Jonathan's comment/callout, everything looks groovy to me! Happy tp approve once that conversation is resolve :-)

**Review by jonathan-Ts — APPROVED** [review]
giphy

**Pull request merged**